src/Glpi/Api/Rest/ItemHandler.php

```diff
@@ -249,4 +251,55 @@ public function deleteItem($itemType, $id, array $inputValues = [], array $query
       ];
    }
 
+   /**
+    * Magic method to execute CRUD actions with a single ItemTypes dynamically.
+    * Send your ItemType as a method and the arguments to be executed.
+    * Each ItemType's name came from the table's name in the DB schema.
+    * Examples:
+    *    Alerts => glpi_alerts,
+    *    Calendars_Holidays => glpi_calendars_holidays,
+    *    Consumableitems => glpi_consumableitems,
+    *
+    * How to use this?. You can check the file 'example/magicCallExample.php' for reference.
+    *
+    * @param string $name of the itemtype
+    * @param mixed $arguments first is method, second is the input, and third is the query string
+    * @return array|null
+    * @throws InsufficientArgumentsException|BadEndpointException|InvalidArgumentException
+    */
+   public function __call($name, $arguments) {
```

Comment:
As you use magic method here, you should add php annotation @ method to help IDE to know **create** **read** **update** and **delete**

This is convenient for maintainers iof the library, but also the developers using it. Exampes are in this diff, where ou deleted these a nnotations.

Reply:
done in commit f7dc8fb70b